src/less/dropdowns.less

```diff
@@ -432,6 +432,7 @@
   z-index: 1;
 }
 
-.multi-select__dropdown__height {
+.multi-select__dropdown__height > div:nth-child(1) {
   max-height: 300px !important;
+  overflow-y: auto !important;
```

Comment:
@yogitattn Is it required to use `overflow-y: auto !important;` here. As I checkout this branch and dropdown is scrolling properly without adding overflow-y property.